pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -1479,8 +1479,11 @@ func (cf *changeFrontier) Start(ctx context.Context) {
 
 	if err := checkpoint.Restore(cf.frontier, cf.spec.SpanLevelCheckpoint); err != nil {
 		if log.V(2) {
-			log.Dev.Infof(cf.Ctx(), "change frontier encountered error on checkpoint restore: %v", err)
+			log.Dev.Infof(cf.Ctx(),
+				"change frontier moving to draining due to error restoring span-level checkpoint: %v", err)
```

Comment:
This seems like it should happen infrequently, should we consider always logging instead of only in verbose logging?

Reply:
I was actually going back and forth on this, I'll change it to be non-verbose

Reply:
Consider making the change in the other err+log+drain locations, too (if you haven't already)

Reply:
Updated, all of the "move to draining logs" are non-verbose now